pkg/controller/submariner/submariner_controller.go

```diff
@@ -392,6 +402,75 @@ func newRouteAgentDaemonSet(cr *submarinerv1alpha1.Submariner) *appsv1.DaemonSet
 	return routeAgentDaemonSet
 }
 
+func newGlobalnetDaemonSet(cr *submarinerv1alpha1.Submariner) *appsv1.DaemonSet {
+	labels := map[string]string{
+		"app":       "submariner-globalnet",
+		"component": "globalnet",
+	}
+
+	matchLabels := map[string]string{
+		"app": "submariner-globalnet",
+	}
+
+	allowPrivilegeEscalation := true
+	privileged := true
+	readOnlyFileSystem := false
+	runAsNonRoot := false
+	security_context_all_cap_allow_escal := corev1.SecurityContext{
+		Capabilities:             &corev1.Capabilities{Add: []corev1.Capability{"ALL"}},
+		AllowPrivilegeEscalation: &allowPrivilegeEscalation,
+		Privileged:               &privileged,
+		ReadOnlyRootFilesystem:   &readOnlyFileSystem,
+		RunAsNonRoot:             &runAsNonRoot,
```

Comment:
In the future, we can identify the minimum caps and try to give only those permission. This is ok for now as we are doing similar things even for other submariner pods.

Reply:
True